setup.py

```diff
@@ -30,6 +30,11 @@
 if __name__ == '__main__':
     requires = ['setuptools_scm', 'numpy', 'xarray >=0.15.1', 'pandas >=1.0.3', 'netCDF4',
                 'h5py', 'pygac >=1.3.1', 'satpy >=0.21.0', 'pyyaml', 'trollsift']
+    extras_require = {
+        "tests": ['cfchecker @ git+https://github.com/cedadev/cf-checker#egg=cfchecker',
+                  # TODO: Replace with cfchecker >= 4.1.0 once released
```

Comment:
Make that an issue rather than a todo, I think it has more visibility

Reply:
Good idea!